### DISCUSSION TITLE
server 404 errors for no reason

### DISCUSSION THREAD

**Discussion by thisisnajafi** [hannoeru/vite-plugin-pages#240] [General]
I have designed my app and made all the components.
In development state it works properly but when I deployed my app on server vite pages plugin does not detect some pages and server redirects them a 404 error page.
How do I solve this problem?

**Comment by jbaubree** [comment]
You have to configure your server to redirect all pages into index.html.
For example with apache:

```
<Directory "/your/app/path">
  Options -MultiViews
  RewriteEngine On
  RewriteBase /
  RewriteRule ^index\.html$ - [L]
  RewriteCond %{REQUEST_FILENAME} !-f
  RewriteCond %{REQUEST_FILENAME} !-d
  RewriteRule . /index.html [L]
</Directory>
```
